libraries/erlang_install.rb

```diff
@@ -81,8 +82,12 @@ def install_dependencies
     # @see Chef::Resource::LanguageInstall#install
     #
     def install
-      build_erlang
-      install_erlang
+      if path_empty?(new_resource.prefix)
+        build_erlang
+        install_erlang
+      else
+        raise "Invalid prefix #{new_resource.prefix}. The prefix must empty or not exist."
```

Comment:
I would reword this to something like "The prefix #{new_resource.prefix} either does not exist or is empty."